lib/internal/event_target.js

```diff
@@ -66,7 +67,7 @@ class Event {
       return name;
 
     const opts = Object.assign({}, options, {
-      dept: options.depth === null ? null : options.depth - 1
+      depth: NumberIsInteger(options.depth) ? options.depth - 1 : options.depth
```

Comment:
I'd assume this was a bug (`dept` instead of `depth`) but wanted to point out since I'm not sure how this interaction works (inspecting circular events I assume?).
Also, use NumberIsInteger instead of just null check.

Reply:
That is both correct. The user should only ever pass through `null` or and integer but it would be possible to pass through other types as well.